opmon/garbage_collection.toml

```diff
@@ -0,0 +1,53 @@
+[project]
+
+name = "Garbage Collection"
+platform = "firefox_desktop"
+xaxis = "submission_date"
+start_date = "2023-09-07"
+skip = false
+is_rollout = false
+skip_default_metrics = true
+compact_visualization = false
+
+metrics = [
+    "gc_ms",
+    "gc_budget_overrun",
+    "gc_effectiveness"
+]
+
+alerts = [
+    "historical_diff"
+]
+
+[project.population]
+
+data_source = "main"
+channel = "release"
```

Comment:
Just FYI, by default this will run on 1% of the release data to save cost

Reply:
So I should remove the next line `monitor_entire_population = true` which would overwrite this default? or is that unconnected?

Reply:
No, that's unrelated. The `main` data source is querying from `main_1pct`: https://github.com/mozilla/metric-hub/blob/3f7fdf3cc993ffd5f6557dd51313a6d43297663b/opmon/defaults/firefox_desktop.toml#L50

So if you really (really) need it then you can override the `main` data source and query from `main_v5` instead. 
Do you think it would make a difference for these metrics?

Reply:
no I really don't need more than 1pct ... just wanted to make sure that I don't accidentally query everything 
thank you very much